src/content/operations/monitoring/_index.en.md

```diff
@@ -20,6 +20,24 @@ The following metrics are exposed currently:
   * `remote_cluster`: the remote cluster name
   * `remote_hostname`: the remote hostname
   * `status`: the connection status (“connecting”, “connected”, or “error”)
+  
+Submariner exposed metrics - the following connectivity metrics shared the same set of labels:
```

Comment:
Sorry for maybe being slow, but this sentence isn't clear to me. What's different between these metrics and the ones above? What does "shared by the same set of labels" mean?

Reply:
@dfarrell07 you're right. I need to find a better way to describe this.
First two metrics exposed by submariner-operator.
There are 5 new metrics - exposed by submariner
4 of them has the same set of labels.
The fifth metric has one more label on top of the previous labels.

Reply:
Humm, yeah that is difficult to express clearly. Maybe some new subsections would help, like an H2 for "Exposed Metrics" and then H3s for "submariner" and "submariner-operator"?

I'm not familiar with the labels, but maybe extracting the "(by cable driver and cable)" bit into the sentence and listing the labels in parentheses would make sense?